src/jit/sideeffects.cpp

```diff
@@ -337,7 +337,7 @@ bool AliasSet::InterferesWith(const AliasSet& other) const
 //    Returns true if the reads and writes in this alias set interfere
 //    with those for the given node.
 //
-//    An alias set interferes with a given node iff it interferes with the
```

Comment:
["iff"](https://en.wikipedia.org/wiki/If_and_only_if) isn't a typo - it means "if and only if".

Reply:
Corrected. Thanks.